fake_switches/cmd/main.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from twisted.internet import reactor
+from fake_switches.switch_configuration import SwitchConfiguration, Port
+from fake_switches.ssh_service import SwitchSshService
+from fake_switches.cisco.cisco_core import CiscoSwitchCore
+
+
+MODEL_MAPPING = {
+    'cisco_generic': CiscoSwitchCore
+}
+
+
+def get_switch_core(model_name):
+    core = MODEL_MAPPING[model_name]
+
+    return core(SwitchConfiguration("127.0.0.1", "my_switch",
+                                    ports=[Port("FastEthernet0/1"),
+                                           Port("FastEthernet0/2"),
+                                           Port("FastEthernet0/3"),
+                                           Port("FastEthernet0/4")]))
+
+
+def main():
+    if len(sys.argv) < 4 or '-h' in sys.argv:
+        print("""fake_switches inline help
+
+fake_switches [-h] model host port
+
+Model can be any of the following:
+    - cisco_generic""")
+        sys.exit(0)
+
+    (model, host, port) = sys.argv[-3:]
```

Comment:
I'm fine with what I wrote (obviously). It's fully tested and it works. Don't forget we can still improve upon this, but I feel this is landable as-is. If you feel otherwise, let me know and I will spend another pomodoro refactoring the arguments parsing :P